modules/openshift-cluster-limits.adoc

```diff
@@ -84,3 +84,6 @@ the system has enough CPU, memory, and disk to satisfy the application requireme
 
 In {product-title} {product-version}, half of a CPU core (500 millicore) is now reserved by
 the system compared to {product-title} 3.11 and previous versions.
+
+In {product-title} 4.1, the number of nodes limit has been lowered until all the scale tests 
+are run at a higher node count.
```

Comment:
Suggested wording change:  In {product-title} 4.1, the tested node limit has been lowered until scale tests can be run at a higher node count.

Reply:
@mffiedler fixed, thanks.